test_manager/features/environment.py

```diff
@@ -98,36 +48,21 @@ def before_all(context):
     :return: None
     """
 
+    context.transport = {}
+    context.ues = {}
+    context.dispatcher = {}
+    context.subprocesses = []
+
     loggerutils.setup_logging()
     loggerutils.setup_formatted_logging(context)
 
-    context.logger.info("Creating Dispatcher...")
-
-    dispatcher = Dispatcher()
-    thread = Thread(target=dispatcher.listen_for_client_connections)
-    thread.start()
-    context.dispatcher = dispatcher
-
-    context.logger.info("Created Dispatcher...")
-    time.sleep(5)
-
-    test_manager = TestManager(context, "127.0.0.5", 12345)
+    test_manager = TestManager(context, "0.0.0.0", 12345)
```

Comment:
does it need to stay as 0.0.0.0?

Reply:
This is changing TM to global ip in case docker wants to connect. You don't have to change TA IPs